idtitlecreatedAtclosedAtclosed
MDU6SXNzdWUxNzEyMDYwMDI=3.2 and 3.3 support?2016-08-15T16:24:46Z2016-08-18T07:51:19ZTrue
MDU6SXNzdWUxNzE0ODEwOTU="Support for explicit retry using ""raise Retry"""2016-08-16T18:08:47Z2016-08-17T08:35:31ZTrue
MDU6SXNzdWUxNzE2MDMyNjU=Provide statistics on the number of retry attempts2016-08-17T08:36:04Z2016-08-25T15:31:01ZTrue
MDU6SXNzdWUxNzE2MDM0NjE=Generate complete documentation of API2016-08-17T08:37:10Z2019-04-18T08:24:38ZTrue
MDU6SXNzdWUxNzE2MDM1ODU=Make sure doc examples run2016-08-17T08:37:55Z2017-09-23T13:16:08ZTrue
MDU6SXNzdWUxNzE2OTE1MzM=Take over any of the 'intersting' functionality of oslo.service.loopingcall2016-08-17T15:43:36Z2016-08-26T07:25:42ZTrue
MDU6SXNzdWUxNzE3NTcwNDc=Discussion: Retry hook for logging or other2016-08-17T20:50:31Z2016-08-17T21:07:41ZTrue
MDU6SXNzdWUxNzIyNzQ4MTc=Get away from using milliseconds everywhere2016-08-20T14:59:49Z2016-08-25T15:38:24ZTrue
MDU6SXNzdWUxNzM2MTU0Nzk=Tests take forever2016-08-27T21:05:53Z2016-08-29T01:28:21ZTrue
MDU6SXNzdWUxNzQ1NzcxMTQ=Retrying compatibility: seconds vs. ms2016-09-01T17:10:22Z2016-09-01T17:44:20ZTrue
MDU6SXNzdWUxNzY0OTcyMDI=retry_if_result fails on any error2016-09-12T22:05:40Z2016-09-13T09:52:11ZTrue
MDU6SXNzdWUxNzY1MDQyMjE=Raise last exception vs always raise RetryError2016-09-12T22:46:08Z2016-09-16T13:24:52ZTrue
MDU6SXNzdWUxNzY1MDUzNzA=Discussion: Test util for mocking time.sleep()2016-09-12T22:53:11Z2016-09-14T08:10:08ZTrue
MDU6SXNzdWUxNzY4NzY1ODg=Implement operator to mimic wait_combine2016-09-14T10:55:53Z2016-10-22T16:59:11ZTrue
MDU6SXNzdWUxOTIzMTI0MTI=Allow to raise custom Exception on failure2016-11-29T15:28:15Z2016-11-30T11:31:15ZTrue
MDU6SXNzdWUxOTM2MDgxNjg=Optionally calling sleep/after on no retry2016-12-05T20:58:02Z2016-12-06T10:03:49ZTrue
MDU6SXNzdWUxOTYwNjIzMjI=Catch exception after retries2016-12-16T13:44:33Z2016-12-17T10:57:25ZTrue
MDU6SXNzdWUxOTg2OTk0MTQ=raising TryAgain when the retrying should be stopped causes deadlock2017-01-04T12:44:28Z2017-01-04T15:26:45ZTrue
MDU6SXNzdWUxOTk4NDM1MTM=Do not use debtcollector2017-01-10T14:59:16Z2017-01-10T15:18:25ZTrue
MDU6SXNzdWUyMDAyMDI0NzY=implement @retry_once and @retry_twice2017-01-11T20:45:15Z2017-01-12T11:33:06ZTrue
MDU6SXNzdWUyMDAzMTUzMjU=Provide a context manager2017-01-12T09:20:58ZFalse
MDU6SXNzdWUyMDExNDU3NDI=weird installation error with 2.7.1: tenacity-3.7.1/setup.cfg: TypeError: dist must be a Distribution instance',)2017-01-17T00:12:49Z2017-03-03T10:42:56ZTrue
MDU6SXNzdWUyMTEzODM3MjE=End without exception2017-03-02T12:57:58Z2017-03-02T13:06:48ZTrue
MDU6SXNzdWUyMTE2NDA4MjQ=add opt in logging2017-03-03T09:47:11Z2017-03-03T10:42:08ZTrue
MDU6SXNzdWUyMTIwNTMzMjY=any plan to support python 2.7?2017-03-06T08:27:57Z2017-03-06T09:19:55ZTrue
MDU6SXNzdWUyMTU1NDMwMDA=`before` callback on first success2017-03-20T20:19:47Z2017-03-21T09:43:59ZTrue
MDU6SXNzdWUyMTY1MTQ0NDE=Required version of Six incorrect2017-03-23T17:41:53Z2017-03-24T09:03:08ZTrue
MDU6SXNzdWUyMTkyMzI0MTU=add to docs how to retrieve the actual function Exception2017-04-04T12:22:18Z2017-04-13T13:24:49ZTrue
MDU6SXNzdWUyMTkzMDU5MjY=Documentation examples don't work2017-04-04T16:18:16Z2017-04-13T13:00:17ZTrue
MDU6SXNzdWUyMjE5MDUzNTg=Provide a generator decorator2017-04-14T22:46:19Z2017-04-16T17:40:30ZTrue
MDU6SXNzdWUyMjM2NDQ2OTM=Retry for Asynchronous Generator (PEP 525)2017-04-23T15:49:38ZFalse
MDU6SXNzdWUyMjM2NDQ3ODQ=Retry for Synchronous Generator2017-04-23T15:51:05ZFalse
MDU6SXNzdWUyMjM2Nzc1NjU=Need to rewind Generators that are passed as argument2017-04-23T23:59:35ZFalse
MDU6SXNzdWUyMjQ1MDk1OTg=run_in_executor_call() in tenacity.AsyncRetrying2017-04-26T15:31:44ZFalse
MDU6SXNzdWUyMzU5MzQ1Nzg=dead code reported by vulture2017-06-14T16:17:39Z2017-06-15T11:31:21ZTrue
MDU6SXNzdWUyMzU5MzkxODQ=How is this package different from retrying?2017-06-14T16:34:13Z2017-06-16T10:03:33ZTrue
MDU6SXNzdWUyMzY3MDA3NTc="Multiple ""stop"" conditions"2017-06-18T05:05:15Z2017-06-20T14:45:09ZTrue
MDU6SXNzdWUyMzY3MDA4ODk=Usage with threaded functions?2017-06-18T05:10:12Z2017-06-21T10:17:08ZTrue
MDU6SXNzdWUyMzY3MDEyOTU=Reset stop conditions2017-06-18T05:23:09Z2017-06-21T10:23:04ZTrue
MDU6SXNzdWUyMzc4MjIxMTQ=Improve on-retry logging2017-06-22T11:59:34Z2018-07-25T12:28:00ZTrue
MDU6SXNzdWUyNDA2NzkxODA=Retries count2017-07-05T14:57:03ZFalse
MDU6SXNzdWUyNDQxMDUxODU=Upload wheels to PyPI2017-07-19T16:47:20Z2017-08-16T14:08:09ZTrue
MDU6SXNzdWUyNDY0MjU2OTI=Changelog?2017-07-28T18:32:21Z2019-04-08T15:27:53ZTrue
MDU6SXNzdWUyNDY4NzgyNzI=Access to elapsed time2017-07-31T20:44:08ZFalse
MDU6SXNzdWUyNDkwODk3MTA=ImportError: No module named 'tasks'2017-08-09T16:33:17Z2017-08-09T20:15:00ZTrue
MDU6SXNzdWUyNTA1NTIwNTQ=Installation issue on macOS-10.12.6 (version 4.4.0)2017-08-16T08:40:02Z2017-08-16T13:06:53ZTrue
MDU6SXNzdWUyNTQwMTczOTg=Can't install Win7 python 2.7.132017-08-30T14:21:42Z2017-09-08T08:26:16ZTrue
MDU6SXNzdWUyNTYwMDI1ODE=Trouble building rpms on Centos 7.32017-09-07T16:48:22Z2019-10-30T12:24:02ZTrue
MDU6SXNzdWUyNjE0NDg4NjE="Broken install:  ""ImportError: No module named 'tasks'"""2017-09-28T20:22:24Z2017-09-28T21:47:20ZTrue
MDU6SXNzdWUyNjc0NzU1ODk="""KeyError: sleep"" in repr(BaseRetrying) and repr(AsyncRetrying)"2017-10-22T15:50:39Z2017-10-24T08:13:53ZTrue
MDU6SXNzdWUyNzI0MzcyMDY=trial number bug when running async function concurrently with retry2017-11-09T04:27:58Z2018-08-27T11:21:49ZTrue
MDU6SXNzdWUyNzI5MDczNDE=retry_on_no_exception2017-11-10T11:47:24Z2017-11-17T14:42:26ZTrue
MDU6SXNzdWUyNzMxNDEzNDQ=Add Tornado support2017-11-11T12:13:10Z2017-11-12T13:31:06ZTrue
MDU6SXNzdWUyNzM5NDg1NzU=Catch specific exception2017-11-14T21:26:05Z2017-11-14T22:08:51ZTrue
MDU6SXNzdWUyNzQ4Nzk0MDY=Tenacity only works with Tornado 4.5 or above2017-11-17T14:20:37Z2018-04-09T14:20:31ZTrue
MDU6SXNzdWUyNzgzMTczMjQ=Ability to have waiters know about prior runs result2017-12-01T00:24:49Z2018-01-13T22:48:08ZTrue
MDU6SXNzdWUyODE1Mjk5Njk='function' object has no attribute 'retry_with'2017-12-12T20:40:47Z2017-12-12T21:14:48ZTrue
MDU6SXNzdWUyODE2MjE5NzQ=Accept or explicitly reject classes derived from BaseException for retrying and py27 compatibility?2017-12-13T04:30:10Z2018-04-09T14:45:41ZTrue
MDU6SXNzdWUyODE5NjcwNTU=New release with new waiter feature :)2017-12-14T03:05:04Z2017-12-14T08:52:03ZTrue
MDU6SXNzdWUyODM1NTUzMzc=Disable `wait` for unittest2017-12-20T13:01:27ZFalse
MDU6SXNzdWUzMDYyMzc3MDE=Documentation page on PyPI has broken formatting2018-03-18T12:52:54ZFalse
MDU6SXNzdWUzMDYyMzc4Njg=Last build on TravisCI failed (version 4.9.0)2018-03-18T12:55:24Z2018-03-22T15:09:44ZTrue
MDU6SXNzdWUzMDk5NTc1OTY=concurrent asyncio retry does not work.2018-03-30T02:22:28Z2018-03-30T02:23:48ZTrue
MDU6SXNzdWUzMTA1MzI2MjI=Cannot run in GAE standard2018-04-02T16:31:06Z2018-04-09T14:46:24ZTrue
MDU6SXNzdWUzMTQzMjgzNDU=syntax error when importing tenacity at python 3.72018-04-14T13:17:18Z2018-04-16T12:05:45ZTrue
MDU6SXNzdWUzMjE2NzczODk=How to catch RetryError?2018-05-09T18:28:21Z2018-05-21T13:20:47ZTrue
MDU6SXNzdWUzMzMwMDg5ODY=Issue with retry_if_exception_type(AssertionError) with python3.6.52018-06-16T19:17:58Z2018-06-19T09:39:13ZTrue
MDU6SXNzdWUzMzk0NjI4NDU=Add retry_if_exception_match2018-07-09T14:02:18ZFalse
MDU6SXNzdWUzNDMxMjIzNTY=SyntaxError for Python 2.72018-07-20T14:10:15Z2018-07-20T15:53:42ZTrue
MDU6SXNzdWUzNDU3NDk4NDQ=Better API for coroutine-returning function?2018-07-30T13:07:12ZFalse
MDU6SXNzdWUzNDg4MDA2MTg=Add min wait for exponential backoff2018-08-08T16:28:30Z2018-08-27T11:22:07ZTrue
MDU6SXNzdWUzNDg5NDE4MDY=Feature suggestion: `after_sleep` keyword argument2018-08-09T00:58:04Z2018-08-17T12:47:32ZTrue
MDU6SXNzdWUzNDk1MTg5NTA=Retry being called despite successful execution of function2018-08-10T13:37:51Z2018-08-17T12:51:55ZTrue
MDU6SXNzdWUzNTE0MDE2NTg=retries not working in a generator context?2018-08-16T23:21:01ZFalse
MDU6SXNzdWUzNTE2NDYwOTI=tenacity 5.0.0 breaks compat2018-08-17T15:49:16Z2018-08-20T13:35:15ZTrue
MDU6SXNzdWUzNTIxMTU3NTA=Add support for Python 3.72018-08-20T12:36:37Z2018-09-10T07:03:25ZTrue
MDU6SXNzdWUzNTIzOTYxNDI=What's new in 5.x?2018-08-21T06:00:01Z2018-08-27T11:22:30ZTrue
MDU6SXNzdWUzNjU2MDA3MzM=Play nice with multiprocessing?2018-10-01T19:07:53ZFalse
MDU6SXNzdWUzNzQwNTMyNTA=How to set default @retry parameters2018-10-25T17:30:29Z2018-10-29T21:51:07ZTrue
MDU6SXNzdWUzNzUzODg0NjU=Access object that contains the retrying function2018-10-30T09:16:30Z2018-11-02T10:39:12ZTrue
MDU6SXNzdWUzODE3MDY5MDE=Documentation example code doesn't work2018-11-16T18:13:45Z2019-04-09T08:10:46ZTrue
MDU6SXNzdWUzODQwNDU3MjI=Retry is not working without decorator2018-11-25T04:10:08Z2018-11-26T12:08:24ZTrue
MDU6SXNzdWUzOTMyMzk4OTI=Retry and unit tests2018-12-20T22:48:44Z2019-01-15T00:46:50ZTrue
MDU6SXNzdWUzOTQ0NjEwNDk=NameError2018-12-27T18:31:49Z2019-05-02T15:56:49ZTrue
MDU6SXNzdWUzOTYyMjIxODE=I am using Python 3.7.1 and successfully installed tenacity with pip.2019-01-06T02:44:23Z2019-01-15T20:02:35ZTrue
MDU6SXNzdWU0MDQwNzYyNTY=getargspec deprecated in python 3.02019-01-29T01:04:22Z2019-01-29T01:11:38ZTrue
MDU6SXNzdWU0MDY5NTA0OTM=stop retrying when a custom exception is raised2019-02-05T19:59:48Z2019-02-14T07:39:08ZTrue
MDU6SXNzdWU0MTExNzg2OTk=how to get func args in retrycallback2019-02-17T11:44:41ZFalse
MDU6SXNzdWU0Mjk3NzUyNjI=`wait_random_exponential(min)`: unexpected behavior 2019-04-05T14:10:42Z2019-04-08T11:38:14ZTrue
MDU6SXNzdWU0MzAwNzAyMDU=retry doesn't pass retry_state to method2019-04-06T19:51:03Z2019-04-08T15:16:23ZTrue
MDU6SXNzdWU0MzMzMTczNzU=Class-based API2019-04-15T14:35:30ZFalse
MDU6SXNzdWU0MzQyMzQ3NzM=Error importing tenacity on python 3.7.32019-04-17T11:36:04Z2019-04-17T12:33:20ZTrue
MDU6SXNzdWU0NDM5NDg1MjI=Exception logging in before_sleep_log looses information on exception printing2019-05-14T14:28:53ZFalse
MDU6SXNzdWU0NDQ3MTgwNjA=Document what statistics dictionary contains2019-05-16T01:55:58ZFalse
MDU6SXNzdWU0NDU5MDE5NDY=Decorators should be higher order functions2019-05-20T02:50:47Z2019-05-20T07:03:31ZTrue
MDU6SXNzdWU0NDk1MTM3NTA=<class 'RuntimeError'>: generator raised StopIteration2019-05-28T22:58:39Z2019-05-31T08:44:14ZTrue
MDU6SXNzdWU0NTUwNDUzNTk=Callback after an attempt (successful or not)2019-06-12T07:05:33Z2019-06-12T12:36:46ZTrue
MDU6SXNzdWU0NTUwNTAxMTY=Different wait strategy for different exceptions2019-06-12T07:19:15Z2019-06-12T12:43:20ZTrue
MDU6SXNzdWU0NTcwMTg5MTQ=wait_exponential has unexpected first delay2019-06-17T16:05:58Z2019-06-21T04:50:45ZTrue
MDU6SXNzdWU0NjkyNDYyMzY=Asyncio: iscoroutinefunction does not work with tenacity2019-07-17T14:23:29Z2019-11-05T14:40:19ZTrue
MDU6SXNzdWU0Njk1NTg3ODA=RetryError cannot be serialized with pickle2019-07-18T03:55:12Z2019-10-28T11:04:25ZTrue
MDU6SXNzdWU0NzEwNTI4ODg=Retrying not working without decorators 2019-07-22T11:54:51Z2019-07-26T08:32:52ZTrue
MDU6SXNzdWU0NzExNzU4NTk=Question: how to get actual Exception?2019-07-22T16:26:22Z2019-07-24T08:28:14ZTrue
MDU6SXNzdWU0NzQ2NDQzMjI=Shold not retry asyncio.CancelledError2019-07-30T15:11:37ZFalse
MDU6SXNzdWU0NzUzMjQ5ODc=Not working with type checker Pyright, https://github.com/Microsoft/pyright2019-07-31T19:46:41ZFalse
MDU6SXNzdWU0Nzk0NDUxNjE=How to use two retry?2019-08-12T02:24:23Z2019-08-13T08:12:55ZTrue
MDU6SXNzdWU0ODE0OTY1OTA=Wheel for 5.1.0 is broken at pypi2019-08-16T08:14:33Z2019-08-16T08:45:47ZTrue
MDU6SXNzdWU0OTI4NzIzOTc=RetryCallState documentation disappears on readthedocs.io2019-09-12T15:17:22ZFalse
MDU6SXNzdWU1MDUxMjI3NDE=Is there any formal way to know the current attempt count and max attempt count in the retry_if_exception callback?2019-10-10T08:40:18ZFalse
MDU6SXNzdWU1MTMzMjQxNzg=local release was 5.1.3 but pypi release was 5.1.42019-10-28T14:02:47Z2019-10-28T14:36:23ZTrue
MDU6SXNzdWU1MTMzMjQzNzQ=Release notes for 5.1.3 and 5.1.42019-10-28T14:03:06Z2019-10-28T14:37:06ZTrue
MDU6SXNzdWU1MTQzMjMyOTU=Missing kwargs in retry helper in __init__.py2019-10-30T00:52:06Z2019-10-31T08:39:16ZTrue
MDU6SXNzdWU1MjI4MTQyNDI=Setup.py missing the package name2019-11-14T12:02:55ZFalse
MDU6SXNzdWU1MjM1MDQ1NTQ=DeprecationWarning re: async def in Python 3.82019-11-15T14:35:05Z2020-02-26T10:37:10ZTrue
MDU6SXNzdWU1Mjk1OTgzNDQ=API reference documentation is broken2019-11-27T22:23:10Z2019-12-16T09:49:03ZTrue
MDU6SXNzdWU1NDY3MTE5ODE=Allow configurable defaults2020-01-08T08:23:41Z2020-01-20T11:04:17ZTrue
MDU6SXNzdWU1NDc1ODQzMTg=Type checking in operators2020-01-09T16:27:00ZFalse
MDU6SXNzdWU1NTI0NzY5NDY=import star is an antipattern2020-01-20T19:38:21ZFalse
MDU6SXNzdWU1ODM2NjA3NDU= In before_log: Starting call to '%s', this is the %s time calling it.2020-03-18T11:28:21ZFalse
MDU6SXNzdWU1ODY1MTIwNjI=Version 6.1.0 returns MagicMock instead of AsyncMock2020-03-23T20:54:00ZFalse
MDU6SXNzdWU1ODc4NDA3NjA='Retrying' object is not iterable2020-03-25T16:51:12Z2020-03-26T10:39:26ZTrue
MDU6SXNzdWU1OTM1MjMxNTE="Adding a ""Decorrelated Jitter"" wait class"2020-04-03T17:10:25ZFalse
MDU6SXNzdWU1OTcxODQyNDc=Regression: Asyncio: iscoroutinefunction does not work with tenacity2020-04-09T10:29:05ZFalse
MDU6SXNzdWU1OTg5OTA2MTA=Reset Retry State2020-04-13T16:33:55ZFalse
MDU6SXNzdWU2MDc4MTE5NjA=Cannot easily mock out sleep function in order to simulate time in tests2020-04-27T19:56:02ZFalse
MDU6SXNzdWU2MDk3NjE1ODA=AsyncRetrying does not work in the latest release on PyPI2020-04-30T09:36:58Z2020-04-30T10:01:21ZTrue
MDU6SXNzdWU2MTA1NjgxNzg=Version 6.2.0 type hinting breaks mypy with --strict2020-05-01T05:13:30ZFalse
MDU6SXNzdWU2MTg2NTM3MzQ=Modify input based on result of attempt2020-05-15T02:32:13ZFalse
MDU6SXNzdWU2MTg3MjAyNTY=retry_error_callback not copied by Retrying.copy2020-05-15T06:01:01ZFalse
